packages/backend/ubuntu-pro.go

```diff
@@ -99,20 +112,27 @@ func isServiceEnabled(obj dbus.BusObject) (bool, error) {
     return status.Value().(string) == "enabled", nil
 }
 
+func isMachineProAttached(manager dbus.BusObject) (bool, error) {
```

Comment:
I would change this to have the pro service as a pointer receiver so you don't have to pass the manager around, and maybe rename it to something like `isAttached` to make to a bit clearer this is just a check and fairly different to `IsMachineProAttached`.

Otherwise I think this makes perfect sense to double check the FE usage of these endpoints :smile:

Reply:
Oh of course that's so much better, refreshing even. Thanks!